go.mod

```diff
@@ -1,37 +1,39 @@
 module github.com/grafana/xk6-websockets
 
-go 1.20
+go 1.21
+
+toolchain go1.22.3
 
 require (
-	github.com/gorilla/websocket v1.5.1
-	github.com/grafana/sobek v0.0.0-20240606091932-2da0e9e5f3e7
+	github.com/gorilla/websocket v1.5.3
+	github.com/grafana/sobek v0.0.0-20240607083612-4f0cd64f4e78
 	github.com/mstoykov/k6-taskqueue-lib v0.1.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
-	go.k6.io/k6 v0.51.1-0.20240606120708-bd114fdbd683
+	go.k6.io/k6 v0.51.1-0.20240612130226-3415ce6edc99
 	go.uber.org/goleak v1.3.0
-	gopkg.in/guregu/null.v3 v3.3.0
+	gopkg.in/guregu/null.v3 v3.5.0
```

Comment:
That could trigger an update in k6 :cry: 

We fixed that in our https://github.com/grafana/k6/blob/master/modtools_frozen.yml#L10-L12 modtools config

Not sure if [we want to update it](https://github.com/grafana/k6/blob/master/Dependencies.md#exceptions), at least probably not from the module :man_shrugging:

Reply:
Ah right! Yeah, I was struggling while finding _"the right balance of updates"_ between `k6` and `sobek` (because the `latest` of each doesn't work due to some incompatibility with the `ast` package), so I ended up updating these by mistake.

Fixed! Thanks! 🙇🏻

Reply:
NP :+1: